moveit_core/constraint_samplers/include/moveit/constraint_samplers/default_constraint_samplers.h

```diff
@@ -69,6 +69,18 @@ class JointConstraintSampler : public ConstraintSampler
   JointConstraintSampler(const planning_scene::PlanningSceneConstPtr& scene, const std::string& group_name)
     : ConstraintSampler(scene, group_name)
   {
+    int rng_seed;
+    if (ros::param::get("~constraint_sampler_random_seed", rng_seed))
+    {
+      ROS_WARN_STREAM_NAMED("constraint_samplers",
+                            "Creating random number generator with seed " << std::to_string(rng_seed));
```

Comment:
No need to convert the int to a string. The streaming operator supports int.
```suggestion
                            "Creating random number generator with seed " << rng_seed);
```